src/main/resources/words_ru.properties

```
@@ -37,3 +37,5 @@ answers.file_reading_error = Извините, произошла ошибка 
 answers.file_saving_error = Извините, произошла ошибка сохранения данных. Пожалуйста, попробуйте начать с команды /start или свяжитесь с разработчиками
 
 answers.invalid_order_exception = Извините, недостаточно данных для расчета или нарушена очередность. Попробуйте начать с команды /start или свяжитесь с разработчиками
+
+answers.tree_reading_error = impossible to read the tree 
```

just 'tree' is too abstract
```suggestion
answers.tree_reading_error = impossible to read the command tree 
```

---

Fixed it, thank you